### PR TITLE
vscode-extensions: update installed extensions

### PR DESCRIPTION
**Pull request by rhoriguchi**
###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by datafoo** [comment]
Each extension change should be in a dedicated commit and have a commit message formatted according to the [documentation](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes) (see examples in the [history](https://github.com/NixOS/nixpkgs/commits/master/pkgs/applications/editors/vscode/extensions/default.nix)).

**Comment by rhoriguchi** [comment]
> Each extension change should be in a dedicated commit and have a commit message formatted according to the [documentation](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes) (see examples in the [history](https://github.com/NixOS/nixpkgs/commits/master/pkgs/applications/editors/vscode/extensions/default.nix)).

Done.

File: pkgs/applications/editors/vscode/extensions/default.nix [hunk 1]
```diff
@@ -1590,8 +1590,8 @@ let
         mktplcRef = {
           name = "java";
           publisher = "redhat";
-          version = "1.3.0";
-          sha256 = "sha256-Y5hP/Rq9BsFwbCRQWOfiLHKoYkKBpZx8blg9o74obfk=";
+          version = "1.4.0";
+          sha256 = "sha256-9q3ilNukx3sQ6Fr1LhuQdjHHS251SDoHxC33w+qrfAI=";
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/vscode/extensions/default.nix:1594]:
```suggestion
          sha256 = "0fzdkwjs79marnj2rzw207s5isgibgw8jrkdqs666w99q65nzjij";
```

Comment (rhoriguchi) [reply pkgs/applications/editors/vscode/extensions/default.nix:1594]: I've updated all hashes to the `sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=` format.

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 163634 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>9 packages built:</summary>  <ul>    <li>vscode-extensions.eamodio.gitlens</li>    <li>vscode-extensions.esbenp.prettier-vscode</li>    <li>vscode-extensions.formulahendry.auto-close-tag</li>    <li>vscode-extensions.formulahendry.auto-rename-tag</li>    <li>vscode-extensions.jock.svg</li>    <li>vscode-extensions.ms-azuretools.vscode-docker</li>    <li>vscode-extensions.pkief.material-icon-theme</li>    <li>vscode-extensions.redhat.java</li>    <li>vscode-extensions.redhat.vscode-yaml</li>  </ul></details>

File: pkgs/applications/editors/vscode/extensions/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/applications/editors/vscode/extensions/default.nix:1594]: That is what ofborg gave me...

Comment (rhoriguchi) [reply pkgs/applications/editors/vscode/extensions/default.nix:1594]: The update script still uses the old format, which output I copy pasted.

Comment (SuperSandro2000) [reply pkgs/applications/editors/vscode/extensions/default.nix:1594]: The format does not matter. Somehow the hash behind it which can have multiple formats, was wrong.

**Pull request merged**